### ISSUE TITLE
Updater

### ISSUE THREAD

**Issue by jaruba** [jaruba/PowderWeb#8]
Although I'd prefer an auto-updater, those can cause a lot of issues. So unless I find a good way to do an auto-updater, I'm fine with an update notification too.

**Comment by marcus-sa** [comment]
I'd suggest checking out https://github.com/electron/update-electron-app

**Comment by jaruba** [comment]
Thanks for the link, usage seems simple enough. I don't particularly enjoy the fact that it requires a Code Signing Certificate, but it's about time I bought one..

**Comment by marcus-sa** [comment]
Yeah, it's pretty damn retarded that Microsoft demands you buy a CSC.. 🤦
But yeah it's sweet that it works out of the box with Github repositories.

**Comment by Vastrix** [comment]
wow, that is indeed bullshit :o
